l10n_it_corrispettivi_fatturapa_out/views/account_invoice_view.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  Copyright 2019 Simone Rubino
+  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+  -->
+
+<odoo>
+    <record id="corrispettivi_fatturapa_out_form" model="ir.ui.view">
+        <field name="name">Integrate corrispettivi and fatturapa out form</field>
```

Comment:
Se qui ti riferisci ai due moduli ti conviene riportare il nome completo, in modo che sia più chiaro.
l10n_it_corrispettivi e l10n_it_fatturapa_out

Altrimenti 
corrispettivi → sums due.

Reply:
Grazie, fatto